httpie/downloads.py

```diff
@@ -60,39 +63,61 @@ def parse_content_range(content_range, resumed_from):
     match = re.match(pattern, content_range)
 
     if not match:
+        has_branched('parse_content_range', 2)
         raise ContentRangeError(
             'Invalid Content-Range format %r' % content_range)
 
     content_range_dict = match.groupdict()
     first_byte_pos = int(content_range_dict['first_byte_pos'])
     last_byte_pos = int(content_range_dict['last_byte_pos'])
-    instance_length = (
-        int(content_range_dict['instance_length'])
-        if content_range_dict['instance_length']
-        else None
-    )
+    
+    if content_range_dict['instance_length']:
+        has_branched('parse_content_range', 3)
+        instance_length = int(content_range_dict['instance_length'])
+    else:
+        has_branched('parse_content_range', 4)
+        instance_length = None
+    
+    if content_range_dict['instance_length']:
+        has_branched('parse_content_range', 5)
+    else:
+        has_branched('parse_content_range', 6)
```

Comment:
This used to check the statement above. Now it's duplicated so this should be removed.